metrics/metrics.go

```diff
@@ -189,3 +189,8 @@ func PrintMeanStd(resultsPath, funcName string, metricsList ...*Metric) error {
 func ToUS(dur time.Duration) float64 {
 	return float64(dur.Microseconds())
 }
+
+// ToUS Converts Duration to milliseconds
+func ToMs(dur time.Duration) int64 {
```

Comment:
```suggestion
// ToMS Converts Duration to milliseconds
func ToMS(dur time.Duration) int64 {
```

don't forget to lint the code